@here/harp-mapview/lib/MapView.ts

```diff
@@ -826,7 +826,8 @@ export class MapView extends THREE.EventDispatcher {
         // Initialization of the renderer
         this.m_renderer = new THREE.WebGLRenderer({
             canvas: this.canvas,
-            antialias: this.m_options.enableNativeWebglAntialias !== false,
+            antialias:
+                this.m_options.enableNativeWebglAntialias !== false || this.pixelRatio <= 2.0,
```

Comment:
When `pixelRatio <= 2.0` is true then the antialias will be enabled, even if `this.m_options.enableNativeWebglAntialias` is false.

Reply:
Also it would be nice to document this new mechanism as users would not understand it before delving into the source code. Places for this could be in MapView's parameters, in the antialias parameters.

Reply:
True, thanks for spotting! Addressed in new patch-set.